rd-kt/rd-gen/src/test/kotlin/com/jetbrains/rd/generator/test/cases/generator/RdGenTest.kt

```diff
@@ -48,9 +49,46 @@ fun main() {
             rdgen.sources *= "C:\\Work\\rd\\rd-gen\\src\\test\\kotlin\\com\\jetbrains\\rd\\generator\\test\\cases\\generator\\entities"
             rdgen.packages *= "com.jetbrains.rd.generator.test.cases.generator.entities"
         }
+        Configuration.UNREAL_MODEL -> {
+            System.setProperty("model.out.src.lib.ue4.csharp.dir", "C:\\Work\\resharper-unreal\\src\\dotnet\\ReSharperPlugin.resharper_unreal\\obj\\model\\Library")
+            System.setProperty("model.out.src.lib.ue4.cpp.dir", "C:\\Work\\resharper-unreal\\src\\cpp\\RiderLink\\Source\\RiderLink\\Private\\RdEditorProtocol")
+            System.setProperty("model.out.src.lib.ue4.kt.dir", "C:\\Work\\resharper-unreal\\src\\rider\\main\\kotlin\\com\\jetbrains\\rider\\model\\RdRiderProtocol")
+
+            System.setProperty("model.out.src.editorPlugin.csharp.dir", "C:\\Work\\resharper-unreal\\src\\dotnet\\ReSharperPlugin.resharper_unreal\\obj\\model\\RdEditorProtocol")
+            System.setProperty("model.out.src.editorPlugin.cpp.dir", "C:\\Work\\resharper-unreal\\src\\cpp\\RiderLink\\Source\\RiderLink\\Private\\RdEditorProtocol")
+
+            System.setProperty("model.out.src.rider.csharp.dir", "C:\\Work\\resharper-unreal\\src\\dotnet\\ReSharperPlugin.resharper_unreal\\obj\\model\\RdRiderProtocol")
+            System.setProperty("model.out.src.rider.kotlin.dir", "C:\\Work\\resharper-unreal\\src\\rider\\main\\kotlin\\com\\jetbrains\\rider\\model\\RdRiderProtocol")
+
+            rdgen.sources *=
```

Comment:
The hard-coded absolute path looks like a bad idea. Maybe extract base dir to a variable if it required.

Reply:
Absolutely agree, but I've briefly looked through the history of this file and it looks like Dima and Pasha used it as their sandbox and used hardcoded paths all over the place.

Reply:
No problem just fix it :)

Reply:
*fixed. Dropped this test all together, because it expects model from UnrealLink plugin